fr/views/helpers/form.rst

```diff
@@ -2307,7 +2307,7 @@ POST. De plus, par défaut, cela générera des inputs ``hidden`` pour le
 
 Par exemple ::
 
-    // Dans Templates/Tickets/index.ctp
+    // Dans Templates/Tickets/index.php
```

Comment:
`Templates` ?? and several items above

Reply:
@sachiko-kame Do case sensitive search for `Templates/`, it might be more such paths which are harder to find.

Reply:
Thank you!

I will review it.

Reply:
I'm struggling. Please wait a little more.

Reply:
https://github.com/cakephp/docs/pull/6525/files

It seemed okay